.circleci/config.yml

```diff
@@ -5,11 +5,16 @@ jobs:
     docker:
       - image: node:15
 
+    when:
+      condition:
+        or:
+          - equal: ["${CIRCLE_BRANCH}", "${STAGING_BRANCH}"]
+          - equal: ["${CIRCLE_BRANCH}", "${PRODUCTION_BRANCH}"]
+      resource_class: large
```

Comment:
I'm not sure that large resource class is available in our tier.

Reply:
It is apparently, I was surprised as well.